x/erc20/keeper/params.go

```diff
@@ -81,12 +55,38 @@ func (k Keeper) setERC20Enabled(ctx sdk.Context, enable bool) {
 	store.Delete(types.ParamStoreKeyEnableErc20)
 }
 
-// setEnableEVMHook sets the EnableEVMHook param in the store
-func (k Keeper) setEnableEVMHook(ctx sdk.Context, enable bool) {
+func (k Keeper) setDynamicPrecompiles(ctx sdk.Context, dynamicPrecompiles []string) {
 	store := ctx.KVStore(k.storeKey)
-	if enable {
-		store.Set(types.ParamStoreKeyEnableEVMHook, isTrue)
-		return
+	bz := make([]byte, 0)
```

Comment:
Nit: we can predefine the size of the slice using the addressLength and the size of the dynamicPrecompiles slice